src/ai/backend/gateway/config.py

```diff
@@ -18,7 +18,7 @@
         t.Key('addr'): tx.HostPortPair,
         t.Key('name'): tx.Slug[2:64],
         t.Key('user'): t.String,
-        t.Key('password'): t.String,
+        t.Key('password'): t.String(),
```

Comment:
Is there a reason that `t.String` is changed to `t.String()`?

Reply:
@adrysn Oh, I didn't intend to do it. It's just some clumsy clean-up of the previous commit